src/services/teams.js

```diff
@@ -0,0 +1,17 @@
+import { client, checkError } from './client';
+
+export async function fetchTeams() {
```

Comment:
Using the supabase HTTP URL totally works but for constancies sake we may want to use the supabase client for this function too